api/models.py

```diff
@@ -145,6 +147,13 @@ class Node(DatabaseModel):
         default_factory=datetime.utcnow,
         description='Timestamp when node was last updated'
     )
+    max_wait_time: Optional[float] = Field(
+        default=24,
+        description='Maximum time in hours to wait for node to get it \
+completed',
+        ge=0,
+        le=24
```

Comment:
Shouldn't these be floats if the type hint says `Optional[float]`? E.g. `default=24.0` or `default=24.`

Reply:
Yes, agreed.